tools/packaging/common/envoy_bootstrap_v2.json

```diff
@@ -207,6 +207,7 @@
   },
   "admin": {
     "access_log_path": "/dev/null",
+    "profile_path": "{{ .config.ConfigPath }}/envoy.prof",
```

Comment:
Is the ConfigPath default /etc/sitio/proxy?
```
tmpfs on /etc/istio/proxy type tmpfs (rw,relatime)
```

Previously we flush it to /var/log, where we don't neither have the concern that if the profile is large. Now there is the risk: if user runs N times, or run long enough, is the pod get OOM killed due to profiling?

Reply:
is /var/log actually cleaned in k8s?

If the pod uses too much disk space it may eventually get evicted by k8s, but it shouldn't cause any crash or non-graceful failure. The profile isn't so large that I am worried, its only about 1mb or so at least in my case? which seems reasonable, we can likely have 1000s without issue.

it is in /etc/istio/proxy by default